style.css

```
@@ -0,0 +1,48 @@
+body {
+    background-color: #D6D6D6;
+    margin: 0 auto;
+    height: 100vh;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+.navBar, .footer {
+    background-color: #0d6efd;
+    color: white;
+    height: 7.5%;
+}
+
+.navBar{
```

```suggestion
.navBar {
```
Wszędzie spacja przed {